updates from policy to TDA page (#4371)
DFE-Digital/get-into-teaching-app: app/views/content/train-to-be-a-teacher/teacher-degree-apprenticeships.md

* Xavier Teach Southeast, in partnership with the University of Sussex
* University of Derby

> [View all courses for the TDA in secondary maths](https://www.gov.uk/find-teacher-training-courses).

Author: ```suggestion
[View all courses for the TDA in secondary maths](https://find-teacher-training-courses.service.gov.uk).
```

* Xavier Teach Southeast, in partnership with the University of Sussex
* University of Derby

> [View all courses for the TDA in secondary maths](https://www.gov.uk/find-teacher-training-courses).

Reviewer: Shouldn't we consider deeplinking here, given we've specified secondary maths in the link text, rather than make people go through the Find questions?

https://find-teacher-training-courses.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=G1&university_degree_status=false&visa_status=false

Note that this link only includes courses open to applications

Would need to check with policy team as to whether a) deeplinking is better (yes?) and if so b) parameters to be used such as whether to restrict to only open courses

Author: I'll check with policy, my worry was that it's not immediately obvious that it's a TDA (says it in the qualification bit but not at the top) so might confuse users. But it's a good shout so will double check.

## Who’s eligible to apply for a teacher degree apprenticeship?
Teacher degree apprenticeships will be open to anyone who would like to gain a degree and become a qualified teacher. This includes, but is not limited to, teaching assistants, school leavers or people looking to change careers.

> The teacher degree apprenticeship is available to candidates who meet specific eligibility requirements.

Author: ```suggestion
You can apply for a place on a teacher degree apprenticeship if you meet specific eligibility requirements.
```

> You can apply for a place on a teacher degree apprenticeship if you meet specific eligibility requirements.

Reviewer: Does this need to say `specific`?

Or could it just say '... if you are eligible. You must be:` and then get rid of line 57 - seems a bit duplicate?

> You can apply for a place on a teacher degree apprenticeship if you meet specific eligibility requirements.

Author: ```suggestion
```

> To be eligiable to apply for a TDA, you must be:

Reviewer: ```suggestion
To be eligible to apply for a TDA, you must be:
```
